### PR TITLE
Clean up image creation

### PR DESCRIPTION
**Pull request by hahnjo**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -19,40 +15,35 @@ jobs:
...
         - alma9
-        - ubuntu2310
-        - ubuntu2304
-        - ubuntu22
+        - fedora38
+        - fedora39
         - ubuntu20
-        - ubuntu18
+        - ubuntu22
+        - ubuntu2304
```
Comment (dpiparo) [inline .github/workflows/build.yml:24]: Isn't 23.04 already past its EOL? If yes, maybe we can get rid of it?

**Review by dpiparo — APPROVED** [review]
Thanks for these changes and the time you dedicated to them.
For me the PR is to be considered approved; I have only a question here: shouldn't we perhaps get rid of ubu23.04? We do not use it for the builds since it's past its EOL.

File: .github/workflows/build.yml (hunk 1 above)
Comment (hahnjo) [reply .github/workflows/build.yml]: Yes indeed, dropped as well :+1:

Comment (dpiparo) [reply .github/workflows/build.yml]: Thanks. Let's merge once the builds are green.

**Pull request merged**